src/strategies/PJON_Strategies.h

```diff
@@ -61,7 +61,8 @@
   #include "ThroughSerial/ThroughSerial.h"
   /* Avoid ATtiny44/84/45/85 missing inclusion error */
   #if !defined(__AVR_ATtiny45__) && !defined(__AVR_ATtiny85__) && \
-      !defined(__AVR_ATtiny44__) && !defined(__AVR_ATtiny84__)
+      !defined(__AVR_ATtiny44__) && !defined(__AVR_ATtiny84__) && \
+      !defined(ESP32)
```

Comment:
ESP32 has no ethernet library (yet)
it is possible to use ethernet modules via serial.
Maybe it will be added later into the ESP32 arduino core.
For now - we have to skip this feature here for ESP32